### ISSUE TITLE
[BUG] Doesn't work with Textual v0.42.0

### ISSUE THREAD

**Issue by davidbrochart** [TomJGooding/textual-slider#39]
### Is there an existing issue for this?

- [x] I have searched the open issues

### Description of the bug

The widget doesn't work with Textual v0.42.0 (it works fine with v0.41.0).

### To Reproduce

1. `pip install "textual>=0.42.0"`
2. Run:
```py
from textual.app import App, ComposeResult
from textual_slider import Slider


class MyApp(App):
    def compose(self) -> ComposeResult:
        yield Slider(0, 10)
        yield Slider(min=200, max=500, step=100, value=300)


if __name__ == "__main__":
    app = MyApp()
    app.run()
```

### Expected behavior

Works with v0.42.0.

### Screenshots

![image](https://github.com/TomJGooding/textual-slider/assets/4711805/441decf4-3b2f-45ec-8149-49d7ae8875cf)

**Comment by TomJGooding** [comment]
Thanks @davidbrochart for reporting this issue! This should now be fixed in the latest version 0.1.2 just released.

**Comment by davidbrochart** [comment]
Thanks a lot!
